### PR TITLE
Update Reference.md

### PR DESCRIPTION
**Pull request by TheDaniMan**
General tips on how to more efficiently clue 1s in early game - I don't know if it belongs in level 10, just that the other levels looked relatively big. It certainly should not go before order chop move.

### CODE REVIEW

**Comment by TheDaniMan** [comment]
Also, is there a way to figure out why the build failed that's less cumbersome than editing and checking? Just going to change this to the original for now, since I don't know if this addition will be accepted in the first place

**Comment by Zamiell** [comment]
>  is there a way to figure out why the build failed

if you click details and then build, it says why it failed:

```
/home/travis/build/Zamiell/hanabi-conventions/Reference.md:1788:103 - Unknown word (bottomdeck)
/home/travis/build/Zamiell/hanabi-conventions/Reference.md:1790:17 - Unknown word (occurence)
The command "./spell_check.sh" exited with 1.
```

**Comment by TheDaniMan** [comment]
Oh wow, I didn't realize it factored in spelling. Anyways, thoughts of having a section like this in the doc? Because it took me quite a lot of games to appreciate the value of saving non-trash that isn't soon to be playable.

**Comment by Zamiell** [comment]
i am lukewarm but its okay i guess

File: Reference.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ Note that convention updates for the group are listed in the `#announcements` ch
   * [Common Mistakes](#level-8---intermediate---common-mistakes)
 * [Level 9 - Fellow](#level-9---fellow)
 * [Level 10 - Journeyman](#level-10---journeyman)
+  * [General Principles](#level-10---journeyman---general-principles)
```
Comment (Zamiell) [inline Reference.md:45]: you forgot to add the existing section here

**Pull request merged**